strapi/helm/strapi/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
Usually it's a good idea to drop these helm tests as they can make deployments more flaky

Reply:
Removing